jsonrpc/jsonrpc_test.go

```diff
@@ -0,0 +1,25 @@
+package jsonrpc
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSendJsonRpcRequest(t *testing.T) {
+	addr := resetMockRelay()
+	req := NewJSONRPCRequest(1, "eth_call", "0xabc")
+	res, err := req.Send(addr)
+	assert.Nil(t, err, err)
```

Comment:
I think one more err is not required
```suggestion
	assert.Nil(t, err, err)
```